UnitsNet.WindowsRuntimeComponent/UnitsNet.WindowsRuntimeComponent.csproj

```diff
@@ -11,21 +11,19 @@
     <AssemblyName>UnitsNet</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
```

Comment:
I assume this change was not intentional? Possibly by just opening the project in visual studio. If so, just revert this file.

Reply:
Yes, it was not intentional.
Thank you for the patience.